package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "mongo-memory-server-action",
   "description": "Support MongoDB-dependent GitHub Workflows",
-  "version": "1.0.2",
+  "version": "1.1.0",
```

Comment:
not really your fault but why are we committing the build directory?

Reply:
This is intentional - the build is committed because GitHub Actions using JavaScript require the transpiled code (the native JavaScript) and we want the merge into the mainline branch to do just one thing: publish a tag with the usable code in it.